src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -1471,30 +1471,30 @@ u16 EnOssan_PartTimer_GetWelcome(EnOssan* this, PlayState* play) {
     }
 }
 
-void EnOssan_SetHaveMet(EnOssan* this) {
+void EnOssan_SetTalked(EnOssan* this) {
```

Comment:
I wonder if this should have `Flag` or `WeekEventReg` in the function name?

Reply:
Hmm, I'm not really sure we need to describe the implementation details in the function name.

Reply:
`SetTalked` just seems a bit weird but maybe it's just me. So it's probably fine